ejb/src/test/java/biz/karms/sinkit/ejb/impl/ArchiveServiceEJBTest.java

```diff
@@ -0,0 +1,108 @@
+package biz.karms.sinkit.ejb.impl;
+
+import biz.karms.sinkit.ejb.ArchiveService;
+import biz.karms.sinkit.ejb.BlacklistCacheService;
+import biz.karms.sinkit.ejb.WhitelistCacheService;
+import biz.karms.sinkit.ejb.cache.pojo.WhitelistedRecord;
+import biz.karms.sinkit.ejb.elastic.ElasticService;
+import biz.karms.sinkit.ejb.elastic.logstash.LogstashClient;
+import biz.karms.sinkit.ejb.util.IoCIdentificationUtils;
+import biz.karms.sinkit.exception.TooOldIoCException;
+import biz.karms.sinkit.ioc.*;
+import org.elasticsearch.index.query.QueryBuilder;
+import org.elasticsearch.search.sort.SortBuilder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+/**
+ * @author Tomas Kozel
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class ArchiveServiceEJBTest {
+
+    //TODO: Shouldn't log messages be also shown (could be useful for debugging)
+    @Mock
+    private Logger log;
```

Comment:
@hugeox It does not make sense to me. It is either a misunderstanding or I am missing something important  to understand it. One mocks those resources that are unavailable or too difficult and unnecessary for the test to setup. Logger is perfectly fine as long as the EJB is tested with Arquillian inside the container and CDI works and provides the Logger resource.   What is the problem with it you are trying to solve?

Reply:
But these unittests are not tested with arquillian.
The logger needs to be mocked, because within the tests there are calls made to it and those fail if logger isn't mocked.

Reply:
O.K. Leave it as it is for now.